### ISSUE TITLE
Replace non-standard annotation by standard annotation attribute `groupImage`

### ISSUE THREAD

**Issue by modelica-trac-importer** [modelica/Modelica#408]
**Reported by dietmarw on 12 Sep 2010 00:22 UTC**
Currently the MSL makes heavy use of the non-standard and tool dependent annotation:

```
annotation(__Dymola_Images(Parameters(group="Foo", 
source=modelica://Modelica/Resources/Images/Foo.png")))
```

Since Modelica 3.2 there exists the new standard annotation attribute `groupImage`, e.g., 
```
annotation(Dialog(group = "Foo",
groupImage="modelica://MyPackage/Resources/Images/Foo.png"))
```
So all the non-standard annotations should be replaced by the use of the standard `Dialog` annotation attribute.

----

Migrated-From: https://trac.modelica.org/Modelica/ticket/408

**Comment by modelica-trac-importer** [comment]
**Modified by dietmarw on 21 Feb 2011 08:21 UTC**

**Comment by modelica-trac-importer** [comment]
**Comment by dietmarw on 7 Mar 2012 14:01 UTC**
Fixed in ce815fa481f006d3c904343e4f9301b4da5dcb34.
